crates/matrix-sdk-appservice/Cargo.toml

```diff
@@ -40,7 +40,8 @@ url = "2"
 warp = { version = "0.3.1", optional = true, default-features = false }
 
 [dependencies.ruma]
-version = "0.4.0"
+git = "https://github.com/ruma/ruma"
+rev = "0101e110f"
```

Comment:
Is there an ETA for when we'll have a release with all the new goodies?

Reply:
No, just a GH milestone: https://github.com/ruma/ruma/milestone/7